recipe/conda_build_config.yaml

```diff
@@ -43,7 +43,7 @@ clang_version:
 
 # switch over release flag style cleanly across compiler versions, see #98
 meson_release_flag:
-  - "--buildtype release"
+  - "-Dbuildtype=release"
   - "-Dbuildtype=release"
   - "-Dbuildtype=release"
```

Comment:
When we did the change-over of the flags (from #98 originally; done in #121), you had wanted to do keep this separate across compiler versions. I guess we're now running into situations where going back to GCC 12 for some unrelated reason causes problems with the meson flags? I'm fine with the change in itself, but what should be done is to remove the whole `meson_release_flag` variable completely. It only existed to serve the transition.

Reply:
I think this was ultimately my mistake in https://github.com/conda-forge/ctng-compiler-activation-feedstock/commit/c8d0da5bd1a5e31ed456b724f10ab78b818050f3, in the sense that the order should have been
```yaml
meson_release_flag:
  - "--buildtype release"  # for GCC 12
  - "-Dbuildtype=release"  # for GCC 13
  - "-Dbuildtype=release"  # for GCC 14
```
but that was the wrong way around for zipping with
```yaml
gcc_version:
  - 14.1.0
  - 13.3.0
  - 12.4.0
```
Sorry about that.